.github/scripts/all_tests.sh

```diff
@@ -3,6 +3,35 @@
 # exit if error
 set -e
 
+
+# check that in scripts directory
+
+CURRENT_DIR="$(pwd)"
+SCRIPTS_DIR='obsidian-hub/.github/scripts'
```

Comment:
This requires that the checked-out directory is called `obsidian-hub`.

My fork is called `obsidian-hub-claremacrae` so this would fail for me.

I've seen scripts achieve this kind of thing by automatically `cd`-ing to the script directory, to allow themselves to be run from any directory...

If you really feel it's worth it, that would be a more general approach.

Reply:
I see, I went from the assumption that someone cloned this repo, but the name could be different, I didn't consider that.

Reply:
Concerning scripts that automatically go to their directory: I thought about that, but then I would need to know from which directory I start.

Reply:
Something like this, although I think I saw a more sophisticated location:
https://stackoverflow.com/questions/3349105/how-can-i-set-the-current-working-directory-to-the-directory-of-the-script-in-ba

Reply:
Thanks!